libkmod/libkmod-index.c

```diff
@@ -401,7 +401,7 @@ void index_dump(struct index_file *in, int fd, const char *prefix)
 		return;
 
 	strbuf_init(&buf);
-	if (strbuf_pushchars(&buf, prefix))
+	if (prefix[0] = '\0' || strbuf_pushchars(&buf, prefix))
```

Comment:
If the API was "completely" broken, this breaks the other half. ;)

This effectively sets the prefix to '\0' instead of checking for it, i.e. `=` vs `==`.

Great work spotting this regression though!

Reply:
Yeah, seems like our dump coverage is worse than I thought.